### PR TITLE
Implemented vulnerability in new platform

### PR DESCRIPTION
**Pull request by hascar**
Implemented vulnerability
The tests are green here: https://ci.openquake.org/job/zdevel_oq-platform2/

### CODE REVIEW

File: openquakeplatform/static/geonode/css/d3.css [hunk 1]
```diff
@@ -98,4 +98,8 @@ svg {
 }
 .grid path {
   stroke-width: 0;
-}
\ No newline at end of file
+<<<<<<< HEAD
+}
+=======
+}
+>>>>>>> origin
```
Comment (nastasi-oq) [inline openquakeplatform/static/geonode/css/d3.css]: Conflict must be fixed.

File: openquakeplatform/settings.py [hunk 2]
```diff
@@ -5,4 +5,4 @@
 import geonode
 
 from geonode.settings import *
-from openquakeplatform.local_settings import *
+#from openquakeplatform.local_settings import *
```
Comment (nastasi-oq) [inline openquakeplatform/settings.py]: If not jet needed remove it.

File: openquakeplatform/static/geonode/css/d3.css (hunk 1 above)
Comment (hascar) [reply openquakeplatform/static/geonode/css/d3.css]: Fixed here: https://github.com/gem/oq-platform2/pull/19/commits/d26a8e6dda9af9e90ba7eedd4c5e9b09e2964ad2

File: openquakeplatform/settings.py (hunk 2 above)
Comment (hascar) [reply openquakeplatform/settings.py]: Fixed here: https://github.com/gem/oq-platform2/pull/19/commits/3a0f5d6ec0cb18202936596ab6b999e188b18c72

File: openquakeplatform/vulnerability/templates/admin/vulnerability/object_history.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
+{% extends "admin/vulnerability/base_site.html" %}
+{% load i18n %}
+{% load admin_urls %}
+
+{% block breadcrumbs %}
+<div class="breadcrumbs">
+mop - object_history
```
Comment (nastasi-oq) [inline openquakeplatform/vulnerability/templates/admin/vulnerability/object_history.html]: text for debug, please remove it

File: openquakeplatform/vulnerability/admin.py [hunk 4]
```diff
@@ -0,0 +1,1163 @@
...
+                _('%(name)s object with primary key %(key)r does not exist.') %
+                {'name': force_unicode(opts.verbose_name),
+                 'key': escape(object_id)})
+
+        using = router.db_for_write(self.model)
+
+        # Populate deleted_objects, a data structure of all related objects
+        # that will also be deleted.
+        (deleted_objects, model_count, perms_needed, protected) = get_deleted_objects(
+        #deleted_objects = get_deleted_objects(
```
Comment (nastasi-oq) [inline openquakeplatform/vulnerability/admin.py]: Please remove it.

File: openquakeplatform/vulnerability/admin.py [hunk 5]
```diff
@@ -0,0 +1,1163 @@
...
+class VU_FuncDistrVulnContInline(OwnedNestedInline, NestedStackedInline):
+    model = FuncDistrVulnCont
+    fk_name = 'vulnerability_func'
+
+
+def formset_by_rel_name(formsets, rel_name):
+    for formset in formsets:
+        #print("prima")
+        #print(formset.fk.related.get_accessor_name())
+        #print("dopo")
```
Comment (nastasi-oq) [inline openquakeplatform/vulnerability/admin.py]: Please remove the three lines above.

File: openquakeplatform/vulnerability/templates/admin/vulnerability/object_history.html (hunk 3 above)
Comment (hascar) [reply openquakeplatform/vulnerability/templates/admin/vulnerability/object_history.html]: Fixed here: https://github.com/gem/oq-platform2/pull/19/commits/b1ceaa2bd408b1e6303a1ced1975e52a7fcfce72

File: openquakeplatform/vulnerability/admin.py (hunk 5 above)
Comment (hascar) [reply openquakeplatform/vulnerability/admin.py]: Fixed here: https://github.com/gem/oq-platform2/pull/19/commits/f440b714cebf6d76ac9232655e33cbfe5237d8a0

File: openquakeplatform/vulnerability/admin.py (hunk 4 above)
Comment (hascar) [reply openquakeplatform/vulnerability/admin.py]: Fixed here: https://github.com/gem/oq-platform2/pull/19/commits/f440b714cebf6d76ac9232655e33cbfe5237d8a0

File: openquakeplatform/vulnerability/templates/admin/vulnerability/base.html [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+    </div>
+    <!-- END Header -->
+    {% block breadcrumbs %}
+    <div class="breadcrumbs">
+    <a href="{% url 'admin:index' %}">{% trans 'Home' %}</a>
+    {% if title %} &rsaquo; {{ title }}{% endif %}
+    </div>
+    {% endblock %}
+{% endif %}
+{# GEM: mn: end #}
```
Comment (nastasi-oq) [inline openquakeplatform/vulnerability/templates/admin/vulnerability/base.html]: all the GEM block above could be removed.

File: pavement.py [hunk 7]
```diff
@@ -16,7 +16,7 @@ def _write_local_settings(lxc_ip, webuiurl, datadir, db_name, db_user, db_pass):
 
     local_settings = open(GEM_LOCAL_SETTINGS_TMPL, 'r').read()
     with open(os.path.join(os.path.expanduser("~"), 'oq-platform2',
-                                                    'openquakeplatform',
+                                                    #'openquakeplatform',
```
Comment (nastasi-oq) [inline pavement.py]: Remove commented app 'openquakeplatform'

File: verifier-guest.sh [hunk 8]
```diff
@@ -26,10 +26,20 @@ GEO_STABLE_HASH="1c65c9b"
 geonode_setup_env()
 {
     export PYTHONPATH=$HOME/oq-platform2:$HOME/oq-platform-taxtweb:$HOME/oq-platform-ipt
-    export DJANGO_SETTINGS_MODULE='openquakeplatform.settings'
+    #export DJANGO_SETTINGS_MODULE='openquakeplatform.settings'
```
Comment (nastasi-oq) [inline verifier-guest.sh]: Remove commented export or comment it with the reason to keep it.

Comment (hascar) [reply verifier-guest.sh]: Fixed here: https://github.com/gem/oq-platform2/pull/19/commits/ef76e94f3ff4e6d8f8f90fc000b84f57588bf9b4

File: pavement.py (hunk 7 above)
Comment (hascar) [reply pavement.py]: Fixed here: https://github.com/gem/oq-platform2/pull/19/commits/94c7034d26f0b60c13872d447c62cd04b4b5e4b7

File: openquakeplatform/vulnerability/templates/admin/vulnerability/base.html (hunk 6 above)
Comment (hascar) [reply openquakeplatform/vulnerability/templates/admin/vulnerability/base.html]: Fixed here: https://github.com/gem/oq-platform2/pull/19/commits/028d9c16b7bc4f10548dcadd5722ca32ec64b9e1

**Pull request merged**